boot/bootdata/readme.txt

```diff
@@ -10,7 +10,7 @@ ReactOS™ is an Open Source effort to develop a quality operating system that i
 compatible with applications and drivers written for the Microsoft® Windows™ NT
 family of operating systems (NT4, 2000, XP, 2003, Vista, Seven).
```

Comment:
Not necessarily true beyond XP (eg : some Vista specific API are not supported, limiting supported applications. Current wording may suggest that any application written for these systems will run in ReactOS

I know it's not the core purpose of the PR but might deserve a fix...

Reply:
> Current wording [...] will [...]

That is worded as a goal, so future, hence all right.
(Although I agree it can be a bit misleading, as current alpha state is obviously not there yet.)

> I know it's not the core purpose of the PR but might deserve a fix...

Rewriting the text is not my purpose at all!
Please, avoid such unrelated comments.